controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -1087,6 +1087,22 @@ fields:
     type: number
     description: TCP Smoothed Round Trip Time (SRTT), in nanoseconds
     cardinalityWarn: avoid
+  - name: OvsMonitorMD[0]
+    type: string
+    description: OVS monitor metadata for the 1st event
+    cardinalityWarn: avoid
```

Comment:
You should also add these into `columns` and `filters` such as:

```
columns:
  - id: OvsMonitorMD0
    name: 1st OVS metadata
    tooltip: OVS monitor metadata for the 1st event
    field: OvsMonitorMD[0]
    filter: ovs_metadata_0
    default: true
    width: 5
    feature: OvsMonitor
...
filters:
  - id: ovs_metadata_0
    name: OVS monitor metadata for the 1st event
    component: text
    hint: Specify a single metadata.
...
```

Reply:
if we want to make somether smarter that concatenate the events, it will require some code changes in the plugin

Reply:
I believe there is still a need for plugin work anyway to handle the new feature if that is accurate optimizing the above can be part of that too ? for now I will see how far this takes me